content/trusted-content/official-images/contributing.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Contributing to Docker Official Images
+description: |
+  This article describes how Docker Official Images are created,
+  and how you can contribute or leave feedback.
+keywords: docker official images, doi, contributing, upstream, open source
+---
+
+Docker, Inc. sponsors a dedicated team that's responsible for reviewing and
+publishing all content in Docker Official Images. This team works in
+collaboration with upstream software maintainers, security experts, and the
+broader Docker community.
+
+While it's preferable to have upstream software authors maintaining their
+Docker Official Images, this isn't a strict requirement. Creating
+and maintaining images for Docker Official Images is a collaborative process. It takes
+place openly on GitHub where participation is encouraged. Anyone can provide
+feedback, contribute code, suggest process changes, or even propose a new
+Official Image.
+
+> **Note**
+>
+> Docker Official Images are an intellectual property of Docker.
```

Comment:
```suggestion
> Docker Official Images are the intellectual property of Docker.
```

What are we trying to communicate here? Is this necessary to state?

Reply:
Not sure about the background for this. I will check if we really need this callout, I'd prefer to remove it to. Just need to check where it came from, so leaving this for a followup.

Reply:
Yeah this one stood out to me as well, I'm sure it's been some request at some point to add it

Reply:
This note was added based on a request from our Legal counsel

Reply:
https://github.com/docker-library/official-images/issues/14346 🙈👀

Reply:
@usha-mandya can you provide more context please so we can respond to given https://github.com/docker-library/official-images/issues/14346 ?

Reply:
@whalelines Sure. I've shared the original email request that came from Legal and copied Joel for guidance on this.